includes/optional-modules/class-rss.php

```diff
@@ -466,9 +585,31 @@ public static function render_content_settings_metabox( $feed_post ) {
 
 		<script>
 			jQuery( document ).ready( function() {
-				jQuery( '#category_include' ).select2();
-				jQuery( '#category_exclude' ).select2();
-				jQuery( '#tag_include' ).select2();
+				jQuery( '.newspack-ajax-taxonomy-select' ).select2( {
+					ajax: {
+						url: ajaxurl,
+						dataType: 'json',
+						type: 'POST',
+						delay: 250,
```

Comment:
Minor nitpick: I think maybe we want a bigger delay here, something like 2 seconds? I don't think the average person is typing more than 4 characters a second, so the delay isn't throttling a lot of ajax calls. It's generating an ajax request on most characters in my testing.

<img width="1598" height="339" alt="Screenshot 2025-07-25 at 3 27 35 PM" src="https://github.com/user-attachments/assets/f7106e97-b13a-48e2-a1bc-67fd032b2b97" />

Reply:
Feels good to me!